dask/dataframe/tests/test_boolean.py

```diff
@@ -29,3 +29,29 @@ def test_ops():
     dd.utils.assert_eq(ds1 | ds2, s1 | s2)
     dd.utils.assert_eq(ds1 & ds2, s1 & s2)
     dd.utils.assert_eq(ds1 ^ ds2, s1 ^ s2)
+
+
+def test_is_monotonic_numeric():
```

Comment:
I think these tests should go in `test_dataframe.py`

Reply:
Done